CMake/ClangCompilers.cmake

```diff
@@ -56,6 +56,9 @@ if(QMC_OMP)
   endif()
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fopenmp")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp")
+  if(ENABLE_ROCM)
```

Comment:
@markdewing Does adding another variable to this check fix the linking error your experiencing?

Reply:
I had offload enabled. If I disable that, then it works.